javascript-source/systems/quoteSystem.js

```diff
@@ -48,7 +48,7 @@
      */
     function saveQuote(username, quote) {
         var newKey = $.inidb.GetKeyList('quotes', '').length,
-            game = ($.getGame($.channelName) != '' ? $.getGame($.channelName) : "Some Game");
+            game = (!$.javaString($.getGame($.channelName)).isBlank() ? $.getGame($.channelName) : "Some Game");
```

Comment:
Codacy has a fix for the issue: [Replace `······game·=·(!$.javaString($.getGame($.channelName)).isBlank()·?·$.getGame($.channelName)·:·"Some·Game")` with `game·=·!$.javaString($.getGame($.channelName)).isBlank()⏎········?·$.getGame($.channelName)⏎········:·"Some·Game"`](https://app.codacy.com/gh/Psychoboy/PhantomBot/pullRequest?prid=9180768)
```suggestion
      game = !$.javaString($.getGame($.channelName)).isBlank()
```